requirements-xpu.txt

```diff
@@ -3,10 +3,9 @@
 
 setuptools < 70.0.0 # IPEX's torch have some dependency. to be removed.
 
+ray >= 2.9
 torch == 2.3.1+cxx11.abi
```

Comment:
I'm afraid it will not find this wheel if we don't use `--extra-index-url`.
I suggest installing this wheel in docker file, and add some comments here

Reply:
I have added the `--extra-index-url` in the Dockerfile.xpu. It works in our local CI.